op-challenger/fault/cmd/main.go

```diff
@@ -1,43 +1,12 @@
 package main
 
 import (
```

Comment:
its kinda weird to import from another `cmd` package

Reply:
definitely, most of this will be deleted though - just a demo / reference for now